docs/checkout-extension-post-purchase.md

```diff
@@ -1,44 +1,54 @@
-# Post-Purchase Checkout Extensions
+# Installing the Post-Purchase Checkout Extensions
 
-Post-Purchase Checkout Extensions allow merchants to display an additional page between the checkout payment page and the thank-you page, prompting the customer to complete some additional action. In the case of this app, the extension asks the customer to submit a review for the product they just bought.
+This document contains steps on how to setup and install the post-purchase checkout extension for the Node.js application that was installed in the previous step. Each step in this document begins with a reason for why we're doing that step (_usually written like this_), then some instructions for the reader to perform.
 
-If you create a post-purchase checkout extension for your app, it will provide an option to display this additional page to any customer of a store that has your app installed.
+### About Post-Purchase Checkout Extensions
 
-If you want to test the Post-Purchase Checkout Extension locally, you must run it separately. Please note this is only necessary when testing locally, since the published extensions are hosted by Shopify.
+Post-Purchase Checkout Extensions allow merchants to **display an additional page between the checkout payment page and the thank-you page**, prompting the customer to complete some additional action. In the case of this app, the extension asks the customer to submit a review for the product they just bought.
+
+_The code for the post-checkout extension used for this repository is in the [checkout-extension](/checkout-extension) directory._
 
 ## Table of contents
 
-- [Checkout Extension Post-Purchase](#checkout-extension-post-purchase)
-  - [Running the checkout extension locally](#running-the-checkout-extension-locally)
-    - [Install dependencies](#install-dependencies)
-    - [Set up environment](#set-up-environment)
-    - [Start the server](#start-the-server)
-    - [Set up the browser extension](#set-up-the-browser-extension)
-  - [Publishing](#publishing)
-  - [Scaffolding New Checkout Extensions](#scaffolding-new-checkout-extensions)
-  - [Additional Resources](#additional-resources)
+1. [Authenticate with the CLI](#1-authenticate-with-the-cli)
+1. [Create a `.env` file](#2-create-env-file)
+1. [Install the `checkout-ui-extension-run` library](#3-install-the-checkout-ui-extension-run-library)
+1. [Update `embeddedAppHost` variable](#4-update-embeddedapphost-variable)
+1. [Register the extension](#5-register-the-extension)
+1. [Push the extension](#6-push-the-extension)
+1. [Publish the extension](#7-publish-the-extension)
+1. [Enable the extension](#8-enable-the-extension)
+1. [Verify the extension](#9-verify-the-extension)
```

Comment:
Do these 1's get updated? Never seen that 😎 🆒

Reply:
yup!